### PR TITLE
docs: Update Travis CI badge and Development server link on README

### LINKED ISSUE #652: Update TravisCI badge and Development server link on README
## Description
As a contributor,
I need README updated with the project deployment status,
so that I can be up to date.

Some links are not up to date, which can lead to confusion, like the Travis CI badge for develop branch, and the Development server link.

### Update [Required]
- [ ] Change Travis CI develop badge link to https://travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop
- [ ] Change development server link to https://mentorship-backend-temp.herokuapp.com/ 

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
30 minutes

### PR DESCRIPTION
**Pull request by Neilblaze**
### Description

- [x] Updated TravisCI badge

- [x] Updated development server link to `https://mentorship-backend-temp.herokuapp.com/`

Fixes #652 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**

- Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
...
-| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://travis-ci.org/systers/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](http://systers-mentorship-dev.eu-central-1.elasticbeanstalk.com/) |
+| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://api.travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](https://mentorship-backend-temp.herokuapp.com/) |
```
Comment (gaurivn) [inline README.md]:
```suggestion
| [develop](https://github.com/anitab-org/mentorship-backend/tree/develop) | [![Build Status](https://travis-ci.org/anitab-org/mentorship-backend.svg?branch=develop)](https://travis-ci.org/anitab-org/mentorship-backend) | [![Codacy Badge](https://api.codacy.com/project/badge/Grade/5181ca06835045d1aea78fdd30fc37d9?branch=develop)](https://www.codacy.com/app/systers/mentorship-backend) | [Development](https://mentorship-backend-temp.herokuapp.com/) |
```

**Review by gaurivn — CHANGES_REQUESTED** [review]

**Comment by Neilblaze** [comment]
@gaurivn  hope this won't dissapoint you anymore 😂

File: README.md (hunk 1 above)
Comment (Neilblaze) [reply README.md]: ok working on it!

**Review by gaurivn — APPROVED** [review]

**Comment by gaurivn** [comment]
> @gaurivn hope this won't disappoint you anymore 😂

I only reviewed the PRs, thank you for following the guidelines and sending this PR.

**Comment by Neilblaze** [comment]
@gaurivn thanks a bunch for guiding me so compassionately :)

**Review by isabelcosta — APPROVED** [review]
Great work @Neilblaze 🎉 
Thank you for your contribution 👏

**Comment by isabelcosta** [comment]
I'll merged this now, since these are documentation changes and I tested the links properly.

image

**Pull request merged**